README.md

```diff
@@ -2,75 +2,111 @@
 
 ## Usage
 
-Stac-server is a RESTful API that returns JSON, see the [documentation](http://stac-utils.github.io/stac-server), or the /api endpoint which is a self-documenting OpenAPI document. Here are some additional tools that might prove useful:
+Stac-server is a web API that returns JSON, see the [documentation](http://stac-utils.github.io/stac-server), or the /api endpoint which is a self-documenting OpenAPI document. Here are some additional tools that might prove useful:
 
-- [JSONView Chrome Extension](https://chrome.google.com/webstore/detail/jsonview/chklaanhfefbnpoihckbnefhakgolnmc?hl=en): Useful for exploring the API in the browser.
 - [pystac-client](https://github.com/stac-utils/pystac-client): A Python client library and CLI for searching a STAC compliant API
 
-
 ## Deployment
 
 This repository contains Node libraries for running the API, along with a [serverless](https://serverless.com/) configuration file for deployment to AWS.
 
 To create your own deployment of stac-server, first clone the repository:
 
+```sh
+git clone https://github.com/stac-utils/stac-server.git
+cd stac-server
 ```
-$ git clone https://github.com/stac-utils/stac-server.git
-$ cd stac-server
-```
-
-There are some settings that should be reviewed and updated as needeed in the [serverless config file](serverless.yml), under provider->environment:
 
-| Name | Description | Default Value |
-| ---- | ----------- | ------------- |
-| STAC_VERSION | STAC Version of this STAC API | 1.0.0 |
-| STAC_API_VERSION | STAC API Version of this STAC API | 1.0.0-beta.1 |
-| STAC_ID | ID of this catalog | stac-server |
-| STAC_TITLE | Title of this catalog | STAC API |
-| STAC_DESCRIPTION | Description of this catalog | A STAC API |
-| STAC_DOCS_URL | URL to documentation | [https://stac-utils.github.io/stac-server](https://stac-utils.github.io/stac-server) |
-| ES_BATCH_SIZE | Number of records to ingest in single batch | 500 |
-| LOG_LEVEL | Level for logging (CRITICAL, ERROR, WARNING, INFO, DEBUG) | INFO |
-| STAC_API_URL | The root endpoint of this API | Inferred from request |
-| ENABLE_TRANSACTIONS_EXTENSION | Boolean specifying if the [Transaction Extension](https://github.com/radiantearth/stac-api-spec/tree/master/extensions/transaction) should be activated | false |
-
-After reviewing the settings, build and deploy the project.
+Copy the [example serverless config file](serverless.yml.example) to a file named `serverless.yml`:
 
+```sh
```

Comment:
naming the file serverless.yml.example and requring users to copy it makes it less likely that a dev will accidentally commit local changes to their serverless file, and that a merge from the upstream will overwrite customer changes.